zeppelin-web/src/app/jobmanager/jobmanager.controller.js

```diff
@@ -98,10 +98,19 @@ function JobmanagerCtrl($scope, websocketMsgSrv, $interval, ngToast, $q, $timeou
       filterValueInterpreter: '*',
       isSortByAsc: true
     };
+    $scope.sortTooltipMsg = 'Switch to sort by desc';
     $scope.jobTypeFilter = jobManagerFilter;
 
     websocketMsgSrv.getNoteJobsList();
 
+    $scope.$watch('filterConfig.isSortByAsc', function (value) {
+      if (value) {
+        $scope.sortTooltipMsg = 'Switch to sort by desc';
+      } else {
+        $scope.sortTooltipMsg = 'Switch to sort by asc';
+      }
+    });
+
```

Comment:
Thanks for your explanation. Yeah I just thought less code lines. As you said the readability is also important. I don't have much strong about this :)